samples/cellular/gnss/sample.yaml

```
@@ -13,3 +13,105 @@ tests:
       - nrf9160dk/nrf9160/ns
       - nrf9161dk/nrf9161/ns
     tags: ci_build sysbuild
+  # Following configurations will be used by the positioning CI integration
+  # job to verify PRs
```

These would fit into one 100 character line.
And an empty line before integration config makes a nice separation from other compilation configs.

```suggestion

    # Following configurations will be used by the positioning CI integration job to verify PRs
```